### DISCUSSION TITLE
Austin on Windows with conda

### DISCUSSION THREAD

**Discussion by spaceman1313** [P403n1x87/austin#230] [Q&A]
I have Anaconda installed on my windows machine.

According to the Austin documentation, the conda install only works on Linux or Mac. Will a conda install not work on Windows?  MUST I use chocolatey or scoop?

**Comment by P403n1x87** [comment]
@spaceman1313 the maintainers of the [austin-feedstock](https://github.com/conda-forge/austin-feedstock) do not provide Windows builds unfortunately, so it is only available via chocolatey/scoop, and from the release page here on GitHub
